test/AssemblySet_basic_test.py

```diff
@@ -7,6 +7,8 @@
 from test.conftest import INFO_LENGTH
 
 
+N_ASSEMBLIES = 2
```

Comment:
assign this "magic number" to a variable so it's a bit more clear what the number refers to when it is used in tests.